src/documentation/programming-guide.md

```diff
@@ -174,20 +168,8 @@ public static void main(String[] args) {
 ```
 
 ```py
-import apache_beam as beam
-
-# python list
-lines = [
-  "To be, or not to be: that is the question: ",
-  "Whether 'tis nobler in the mind to suffer ",
-  "The slings and arrows of outrageous fortune, ",
-  "Or to take arms against a sea of troubles, "
-]
-
-# Create the pipeline.
-p = beam.Pipeline()
-
-collection = p | 'ReadMyLines' >> beam.Create(lines)
+% github_sample /apache/beam/blob/master/sdks/python/apache_beam/examples/snippets/snippets.py tag:model_pcollection
```

Comment:
Worth doing a quick diff of the generated content to make sure these changes look as expected?